datafiles/templates/Html/noscript-search-form.html.st

```diff
@@ -23,6 +23,7 @@
         <option $if(tags)$        selected $endif$ value=tags>tags</option>
         <option $if(lastUpload)$  selected $endif$ value=lastUpload>date of last upload</option>
         <option $if(maintainers)$ selected $endif$ value=maintainers>maintainers</option>
+        <option $if(packageRank)$ selected $endif$ value=packageRank>packageRank</option>
```

Comment:
the text inside the option shouldn't be camelCase, since it is the human readable description of the column, shown in a drop-down

Reply:
Thank you for the fixes. I hope 7304a05 resolves the last two.